src/app/components/Application/Application.jsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
 
 import { Header, navConfig } from '@nypl/dgx-header-component';
 import Footer from '@nypl/dgx-react-footer';
-import SearchContainer from '../SearchContainer/SearchContainer';
 
 class Application extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.props = props;
```

Comment:
You shouldn't need to do this. The `props` in `this` are already what's coming in from the constructor.